typegate/tests/e2e/typegraph/typegraphs/python/multiple_runtimes.py

```diff
@@ -0,0 +1,29 @@
+from typegraph_next import t, typegraph, Graph
+from typegraph_next.policy import Policy
+from typegraph_next.runtimes.python import PythonRuntime
+from typegraph_next.runtimes.deno import DenoRuntime
+
```

Comment:
not really e2e maybe a generic typegraph folder in test should be used  instead?